makeflow/src/makeflow.c

```diff
@@ -1596,7 +1614,11 @@ int main(int argc, char *argv[])
 				catalog_reporting_on = 1; //set to true
 				break;
 			case 'o':
+#ifdef MPI
```

Comment:
There needs to be a way for every makeflow worker to have their own debug log. Since MPI starts every program with the exact same arguments, it makes sense for the user to pass in the debug file prefix as the output name, and then each worker, including the master, can add on their own rank. The only way to do this is to only capture the prefix name when the user passes it in using `-o`. But makeflow currently starts the debug file process as soon as it parses the `-o` input, and we don't want that in the MPI case. I was trying to go for: "if we're not compiling with MPI, then the code shouldn't compile differently than if the MPI code was never there" and that involved wrapping the debugging in an ifdef